hack/test-cmd.sh

```diff
@@ -333,13 +333,13 @@ osc delete imageRepositories test
 echo "imageRepositoryMappings: ok"
 
 [ -n "$(osc get imageRepositories mysql -t "{{ index .metadata.annotations \"openshift.io/image.dockerRepositoryCheck\"}}")" ]
-[ "$(osc new-app php mysql -o yaml | grep 3306)" ]
+[ "$(osc new-app mysql -o yaml | grep 3306)" ]
 # verify we can generate a Docker image based component "mongodb" directly
 [ ! "$(osc new-app unknownhubimage -o yaml)" ]
 [ "$(osc new-app mongo -o yaml | grep library/mongo)" ]
 # the local image repository takes precedence over the Docker Hub "mysql" image
 [ "$(osc new-app mysql -o yaml | grep mysql-55-centos7)" ]
-osc new-app php mysql
+osc new-app mysql
```

Comment:
We shouldn't generate a service for PHP if it exposes no ports.

----- Original Message -----
> >  # verify we can generate a Docker image based component "mongodb" directly
> >  [ ! "$(osc new-app unknownhubimage -o yaml)" ]
> >  [ "$(osc new-app mongo -o yaml | grep library/mongo)" ]
> >  # the local image repository takes precedence over the Docker Hub "mysql"
> >  image
> >  [ "$(osc new-app mysql -o yaml | grep mysql-55-centos7)" ]
> > -osc new-app php mysql
> > +osc new-app mysql
> 
> Here's the error:
> ```
> Error: Service "php" is invalid: spec.ports: required value
> ```
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1747/files#r29058635